queries.sql

```diff
@@ -1,3 +1,12 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+-- MONDAY
+
+SELECT * FROM animals WHERE name LIKE '%mon';
+SELECT name FROM animals WHERE date_of_birth BETWEEN '2016-01-01' and '2019-01-01';
```

Comment:
- [ ] in this query you suppose to include the whole year of 2019, not the first day in it only. So please edit this query.
[Hint] You can do it like that:
```sql
SELECT name FROM animals WHERE date_of_birth BETWEEN '2016/01/01' and '2019/31/12'
```
or
```sql
SELECT name FROM animals WHERE EXTRACT(year FROM date_of_birth) BETWEEN 2016 AND 2019
```

Reply:
Changed! ✔️ Thanks!